.editorconfig

```diff
@@ -0,0 +1,26 @@
+# https://pinterest.github.io/ktlint/latest/rules/configuration-intellij-idea/
+
+root = true
+
+[*]
+insert_final_newline = true
+
+[{*.kt,*.kts}]
+# Android Studio standard
+max_line_length = 120
+ij_kotlin_code_style_defaults = KOTLIN_OFFICIAL
+ij_kotlin_allow_trailing_comma = true
+ij_kotlin_allow_trailing_comma_on_call_site = true
+#  Disable wildcard imports entirely
+ij_kotlin_name_count_to_use_star_import = 2147483647
```

Comment:
:thinking: ~~Could you quickly explain what these magic numbers mean?~~ Just saw that this is just taken from ktlin config, still looks weird. :D

Reply:
Yes, apparently, there is no better way to disable this.